### PR TITLE
copy get-started part in readme into website

### PR DESCRIPTION
**Pull request by liadbiz**
detail:

* what: add getting-started documentation into glamorus website
* why: fill documentation website!
* how: copy content in readme.md file in glamorous repo into
getting-started.js

### CODE REVIEW

File: pages/basics/content/getting-started.js [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
   title: 'Getting Started',
-  subtitle: '',
-  description: ``.replace(/~/g, '`'),
+  subtitle: 'start using Glamorous',
+  description: `
+    <a href="https://youtu.be/lmrQTpJ_3PM" title="glamorous walkthrough">
+    <img src="https://github.com/paypal/glamorous/raw/master/other/glamorous-walkthrough.png" alt="Video Screenshot" title="Video Screenshot" width="700" />
+  </a>
+
+  Here are a basic example that show you how to get started making component with ~css-in-js~ magic
```
Comment (tdeschryver) [inline pages/basics/content/getting-started.js]: Shouldn't it be better if this was being written as:
Here is a basic example that shows you how to get started making a component with `css-in-js` magic.

Comment (paulmolluzzo) [inline pages/basics/content/getting-started.js]: Agreed, that reads better.

Comment (liadbiz) [inline pages/basics/content/getting-started.js]: Thanks! will correct it soon.

**Comment by kentcdodds** [comment]
Could you take a screenshot of what this looks like? Thanks!

**Comment by liadbiz** [comment]
@kentcdodds  here it is.
image

**Comment by kentcdodds** [comment]
Great! Thanks!

**Pull request merged**